setup.py

```diff
@@ -413,7 +426,7 @@ def not_exists_or_empty(folder):
             os.path.isdir(folder) and len(os.listdir(folder)) == 0
         )
 
-    if bool(os.getenv("USE_SYSTEM_LIBS", False)):
+    if os.getenv("USE_SYSTEM_LIBS"):
```

Comment:
Again, this is unrelated to the PR in question and also kind of semantically broken (it should use `str2bool` or something helper) that would convert 0 and "false" strings to false